recipe/meta.yaml

```diff
@@ -258,11 +265,11 @@ outputs:
       files:
         - test-r-xgboost.r
       commands:
-        - Rscript test-r-xgboost.r
+        - Rscript test-r-xgboost.r  # [not (win and cuda_compiler != "None")]
```

Comment:
To track this, have filed issue: https://github.com/conda-forge/xgboost-feedstock/issues/205